README.md

```diff
@@ -215,12 +234,13 @@ command.
 
 1. Install [NCCL 2](https://developer.nvidia.com/nccl).
 
-If you aren't able to install NCCL 2 Debian package due to missing dependencies, you can use this workaround:
+Steps to install NCCL 2 are listed [here](http://docs.nvidia.com/deeplearning/sdk/nccl-install-guide/index.html).
+
+If you have installed NCCL 2 using `nccl-<version>.txz` package, you should add library path to `LD_LIBRARY_PATH`
+environment variable or register it in `/etc/ld.so.conf`.
```

Comment:
Nit: Can we have NCCL installation steps at one place only, so that we can avoid duplicate & does not have to worry about steps going out of sync? Similar comment on other duplicated instructions for installation

Reply:
Agreed, seems that separate docs folder would solve this and the other comment below.